packages/jest-worker/README.md

```diff
@@ -43,9 +43,7 @@ export function hello(param) {
 
 Node 10 shipped with [worker-threads](https://nodejs.org/api/worker_threads.html), a "threading API" that uses SharedArrayBuffers to communicate between the main process and its child threads. This experimental Node feature can significantly improve the communication time between parent and child processes in `jest-worker`.
 
-We will use worker threads where available. To enable in Node 10+, run the Node process with the `--experimental-worker` flag.
-
-You can explicitly opt-out of this by passing `disableWorkerThreads: true`.
+Since `worker_threads` are considered experimental in Node, you have to opt in to this behavior by passing `enableWorkerThreads: true` when instantiating the worker. While the feature was unflagged in Node 11.7.0, you'll need to run the Node process with the `--experimental-worker` flag for Node 10.
```

Comment:
is it "opt in to" or "opt into"? English is hard, man

Reply:
`opt-in` is fine here.

Reply:
thanks!